packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -250,11 +353,80 @@ class AuroraMysqlClusterEngine extends MySqlClusterEngineBase {
   }
 }
 
+/**
+ * The versions for the Aurora PostgreSQL cluster engine
+ * (those returned by {@link DatabaseClusterEngine.auroraPostgres}).
+ */
+export class AuroraPostgresEngineVersion {
+  /** Version "9.6.3". */
+  public static readonly VER_9_6_3 = AuroraPostgresEngineVersion.of('9.6.3', '9.6');
+  /** Version "9.6.6". */
+  public static readonly VER_9_6_6 = AuroraPostgresEngineVersion.of('9.6.6', '9.6');
```

Comment:
Same question about cut-off here. These are marked as deprecated. Why should they be included?

Reply:
> Same question about cut-off here. These are marked as deprecated. Why should they be included?

Out of curiosity, where did you find that those are deprecated?

Reply:
https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Updates.20180305.html

Reply:
Sweet, thanks! Will definitely remove the deprecated ones.